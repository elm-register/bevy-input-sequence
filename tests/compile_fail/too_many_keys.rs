use bevy_input_sequence::*;

fn main() {
    let _ = key!(A B);
}
