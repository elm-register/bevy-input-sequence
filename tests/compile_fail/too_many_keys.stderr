error: Too many tokens; use keyseq! for multiple keys
 --> tests/compile_fail/too_many_keys.rs:4:20
  |
4 |     let _ = key!(A B);
  |                    ^
